djangocms_navigation/views.py

```diff
@@ -55,31 +57,27 @@ def get_data(self):
         except (TypeError, ValueError):
             pk = None
 
-        if site:
-            if hasattr(model.objects, "on_site"):
```

Comment:
I have had to change this as previous implementation was causing an error. This is because even when the content type is a `Page`, the queryset returned on line 49 (`queryset = versionable_item.grouper_choices_queryset()`) is a standard `QuerySet` instance rather than a `PageQuerySet` instance.

[The `on_site` method exists on `PageQueryset`](https://github.com/django-cms/django-cms/blob/ff6cb9b5dced92eadef62694e989d601e9475b30/cms/models/query.py#L10), which would result in the conditional returning true. But then actual call was being made on a standard `QuerySet` instance where `on_site` does not exist, resulting in a `AttributeError`.